cmd/agent/install_script.sh

```diff
@@ -214,20 +220,29 @@ elif [ $OS = "SUSE" ]; then
     SUSE11="yes"
   fi
 
-  echo -e "\033[34m\n* Installing YUM Repository for Datadog\n\033[0m"
-  $sudo_cmd sh -c "echo -e '[datadog]\nname=datadog\nenabled=1\nbaseurl=https://yum.${repo_url}/suse/${dd_agent_dist_channel}/${dd_agent_major_version}/${ARCHI}\ntype=rpm-md\ngpgcheck=1\nrepo_gpgcheck=0\ngpgkey=https://yum.${repo_url}/DATADOG_RPM_KEY.public\n       https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public' > /etc/zypp/repos.d/datadog.repo"
-
   echo -e "\033[34m\n* Importing the Datadog GPG Keys\n\033[0m"
   if [ "$SUSE11" == "yes" ]; then
+    # SUSE 11 special case
     $sudo_cmd curl -o /tmp/DATADOG_RPM_KEY.public https://yum.${repo_url}/DATADOG_RPM_KEY.public
     $sudo_cmd rpm --import /tmp/DATADOG_RPM_KEY.public
```

Comment:
This could also be wrapped by a conditional to check if we're installing Agent 7:
```bash
  if [ $dd_agent_major_version -ne 7 ]; then
    $sudo_cmd curl -o /tmp/DATADOG_RPM_KEY.public https://yum.${repo_url}/DATADOG_RPM_KEY.public
    $sudo_cmd rpm --import /tmp/DATADOG_RPM_KEY.public
    gpgkeys="https://yum.${repo_url}/DATADOG_RPM_KEY.public\n       https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public"
  else
    gpgkeys="https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public"
  fi
```

Reply:
I was trying to avoid so many nested ifs, I think I managed to simplify it a bit in my last commit.